dev/tools/svg2dart/README.md

```diff
@@ -0,0 +1,5 @@
+# svg2dart
+
+A tool for converting SVG sequences as provided by our motion designers into a Dart representation.
```

Comment:
"our" is ambiguous here. Maybe "A tool for converting SVG sequences generated from After Effects into a Dart representation"?

Reply:
done